### DISCUSSION TITLE
BYD seal support

### DISCUSSION THREAD

**Discussion by Mandrak4** [kommuai/bukapilot#134] [General]
Hi, I'm from Brazil and recently discovered kommu.ai, do you have plans to make it compatible with the model? I saw on the website that atto 3 is compatible so maybe it should work with seal in some degree...

**Comment by a375891918** [comment]
Yes ， i need it too.

**Comment by regisksc** [comment]
I have a BYD Seal from Brazil, also interested

**Comment by silveira2974** [comment]
I need too.

**Comment by sixtyfilms-cmyk** [comment]
also need for BYD seal U. that would be nice...
